<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Techdata</class>
 <widget class="QDialog" name="Techdata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Technical Data</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>tuner_knob.png</normaloff>tuner_knob.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pico Tuner - Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="beaconBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>241</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rx1ParmsBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>171</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rx2ParmsBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>130</y>
     <width>171</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pico Tuner Beacon</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rx1 Parmeters</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rx2 Parmeters</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
